src/olcs/FeatureConverter.js

```diff
@@ -176,9 +176,20 @@ class FeatureConverter {
         })
       });
     } else {
-      primitive.appearance = new Cesium.PerInstanceColorAppearance(options);
+      primitive.appearance = new Cesium.MaterialAppearance({
+        options,
+        material: new Cesium.Material({
+          translucent: color.alpha !== 1,
+          fabric: {
+            type: 'Color',
+            uniforms: {
+              color,
+            }
+          }
+        })
+      });
+      primitive.shadows = 1;
```

Comment:
Casting shadows have performance implications.
So I would have it opt-in (like in CesiumJS).

Let's set it if one of the following properties is set:
- layer.get('olcs_shadows')
- feature.get('olcs_shadows')